hstream-sql/hstream-sql.cabal

```diff
@@ -37,8 +37,9 @@ flag hstream_use_v2_engine
 
 common shared-properties
   ghc-options:
-    -Wall -Wcompat -Widentities -Wincomplete-record-updates
-    -Wincomplete-uni-patterns -Wpartial-fields -Wredundant-constraints
+    -Wall -Wextra -Werror -Wno-name-shadowing -Wno-orphans -Wcompat
```

Comment:
Why turn off these warnings(e.g., name-shadowing) at the package level? Prefer to turn it off at the module level if you really want.